tests/Behat/features/archive-other.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Achive admin other tab
```

Comment:
```suggestion
Feature: Archive admin other tab
```
Super minor but could affect searching for "archive admin" across all files in the future so I figured I'd point it out. Won't hold back merging for it.

Reply:
Updated